ivy/array/creation.py

```diff
@@ -79,6 +79,67 @@ def full_like(
             an array having the same shape as ``self`` and where every element is equal
             to ``fill_value``.
 
+        Functional Examples
+    -------------------
+        With int datatype:
+            >>> x = ivy.Array(ivy.native_array([1,2,3]))
+            >>> fill_value = 0
+            >>> ivy.full_like(x , fill_value)
+            >>> ivy.array([0, 0, 0])
+
+            ivy.array([1, 1, 1, 1, 1, 1])
+            >>> fill_value = 0.000123
+            >>> x = ivy.array(ivy.ones(5))
+            >>> y = ivy.full_like(x , fill_value)
+            >>> print(y)
+            ivy.array([0.000123, 0.000123, 0.000123, 0.000123, 0.000123])
+
+        with float datatype:
+
+            >>> import ivy
+            >>> x = ivy.array([1.0, 2.0, 3.0, 4.0, 5.0, 6.0]) # for filling values with floating datatype the array is required to be of same dtype (float in this case)
+            >>> fill_value = 0.000123
+            >>> y = ivy.full_like(x, fill_value)
+            >>> print(y)
+            ivy.array([0.000123, 0.000123, 0.000123, 0.000123, 0.000123, 0.000123])
+
+        with torch.Tensor input:
+
+            # ivy.full_like will give you output as ivy array
+            >>> import ivy
+            >>> import torch
+            >>> x = torch.Tensor([1.0, 2.0, 3.0, 4.0, 5.0, 6.0])
+            >>> fill_value = 0.001
+            >>> y = ivy.full_like(x, fill_value)
+            >>> print(y)
+            ivy.array([0.001, 0.001, 0.001, 0.001, 0.001, 0.001])
+
+        With ivy.NativeArray input:
+
+            >>> x = ivy.native_array([3.0, 8.0])
+            >>> fill_value = 0.000123
+            >>> y = ivy.full_like(x,fill_value)
+            >>> print(y)
+            ivy.array([0.000123, 0.000123])
+            >>> x2 = ivy.native_array([[3., 8., 2.],
+                                        [2., 8., 3.]])
+            >>> y = ivy.full_like(x2 , fill_value)
+            >>> print(y)
+            ivy.array([[0.000123, 0.000123, 0.000123],
+           [0.000123, 0.000123, 0.000123]])
+
+        Instance Method Examples:
```

Comment:
you can change the title above to instance method examples and remove this part as they will be identical